fire/SFMainMod.F90

```diff
@@ -288,306 +344,126 @@ subroutine CalculateSurfaceRateOfSpread(currentSite)
-             ! To Do: Connect here with the Li & Levis GDP fire suppression algorithm. 
-             ! Equation 16 in arora and boer model JGR 2005
-             ! AB = AB *3.0_r8
-
-             !size of fire = equation 14 Arora and Boer JGR 2005 (area of an ellipse)
-             size_of_fire = ((pi_const/(4.0_r8*lb))*((df+db)**2.0_r8))
-
-             ! AB = daily area burnt = size fires in m2 * num ignitions per day per km2 * prob ignition starts fire
-             ! AB = m2 per km2 per day
-             ! the denominator in the units of currentSite%NF is total gridcell area, but since we assume that ignitions 
-             ! are equally probable across patches, currentSite%NF is equivalently per area of a given patch
-             ! thus AB has units of m2 burned area per km2 patch area per day
-             AB = size_of_fire * currentSite%NF * currentSite%FDI
-
-             ! frac_burnt 
-             ! just a unit conversion from AB, to become area burned per area patch per day, 
-             ! or just the fraction of the patch burned on that day
-             currentPatch%frac_burnt = (min(0.99_r8, AB / km2_to_m2))
-             
-             if(write_SF == itrue)then
-                if ( hlm_masterproc == itrue ) write(fates_log(),*) 'frac_burnt',currentPatch%frac_burnt
-             endif
-
-          else
-             currentPatch%frac_burnt = 0._r8
-          endif ! lb
-
-         ROS   = currentPatch%ROS_front / 60.0_r8 !m/min to m/sec 
-         W     = currentPatch%TFC_ROS / 0.45_r8 !kgC/m2 of burned area to kgbiomass/m2 of burned area
-
-         ! EQ 15 Thonicke et al 2010
-         !units of fire intensity = (kJ/kg)*(kgBiomass/m2)*(m/min)
-         currentPatch%FI = SF_val_fuel_energy * W * ROS !kj/m/s, or kW/m
-       
-         if(write_sf == itrue)then
-             if( hlm_masterproc == itrue ) write(fates_log(),*) 'fire_intensity',currentPatch%fi,W,currentPatch%ROS_front
-         endif
-
-         !'decide_fire' subroutine 
-         if (currentPatch%FI > SF_val_fire_threshold) then !track fires greater than kW/m energy threshold
-            currentPatch%fire = 1 ! Fire...    :D
-            !
-            currentSite%NF_successful = currentSite%NF_successful + &
-                 currentSite%NF * currentSite%FDI * currentPatch%area / area
-            !
-         else     
-            currentPatch%fire       = 0 ! No fire... :-/
-            currentPatch%FD         = 0.0_r8
-            currentPatch%frac_burnt = 0.0_r8
-         endif         
-          
-       endif ! NF ignitions check
-       endif ! nocomp_pft_label check
-       
-       currentPatch => currentPatch%younger
-
-    enddo !end patch loop
-
-  end subroutine area_burnt_intensity
+          length_to_breadth = LengthToBreadth(currentSite%fireWeather%effective_windspeed, tree_fraction_patch)
 
+          ! fire size [m2]
+          fire_size = FireSize(length_to_breadth, currentPatch%ROS_back,              &
```

Comment:
Unnecessary spacing before `&` here.

Reply:
This is a style thing... maybe it's just me. I try to put the `&` at a specific row, but I can discontinue that

Reply:
Ah ok. I generally try not to do alignment stuff like that because it inevitably gets messed up or needs to be re-aligned, but if that's your style I won't fight you on it.